.github/workflows/stale.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           stale-pr-label: " "
 
       - name: Print outputs for issues
-        run: echo ${{ join(steps.stale-issues.outputs.*, ',') }}
```

Comment:
Is this change related to the `set-output` deprecation?

Reply:
I see now this is from https://github.com/actions/stale/pull/859

Reply:
Not is related, but since it is a superfluous change, I have taken advantage of this PR to do it.

It should be do it past in #9242 bump